lib/localparticipant.js

```diff
@@ -129,12 +114,6 @@ class LocalParticipant extends Participant {
 
     const signaling = this._signaling;
 
-    function localTrackAdded(localTrack) {
-      signaling.addTrack(localTrack._trackSender, localTrack.name);
-      log.info(`Added a new ${util.trackClass(localTrack, true)}:`, localTrack.id);
-      log.debug(`${util.trackClass(localTrack, true)}:`, localTrack);
-    }
-
```

Comment:
@markandrus Since "trackAdded" and "trackRemoved" events have been removed, we call those methods in `publishTrack` and `unpublishTrack` methods respectively. Hence the conversion to private methods.